SLMM.Api/Requests/TurnRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SLMM.Api.Requests
+{
+    public class TurnRequest
+    {
+        public short Degrees { get; set; }

```

Comment:
Only 90 and -90 degrees are allowed right now. See LawnMowingMachine. Although in the near future, other angles might be supported.
Now that i think of, maybe a better approach would have been to define an enum Direction: Clockwise, Anticlockwise., in order to avoid negative values.